<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorLegendEditor</class>
 <widget class="QDialog" name="ColorLegendEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>388</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>388</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>275</width>
    <height>388</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图例编辑器</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>256</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>添加图例</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editButton">
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
